src/containers/pages/OrganizationViews/SingleOrganizationView/index.css

```diff
@@ -0,0 +1,7 @@
+.link-as-button .btn {
+  float: right;
+}
+
+div.row div.col div.row div.col .col-6 {
```

Comment:
What is this style affecting? We should probably add an actual class if at all possible.